CorpusSearch/Dockerfile

```diff
@@ -0,0 +1,33 @@
+﻿FROM mcr.microsoft.com/dotnet/aspnet:8.0 AS base
+RUN apt-get update && apt-get upgrade -y && \
+    apt-get install -y nodejs \
+    npm  
```

Comment:
The base image shouldn't need Node, it should only be part of the build step

Reply:
Will move this, I guess